src/material/tooltip/tooltip.ts

```diff
@@ -232,7 +232,7 @@ export class MatTooltip implements OnDestroy, OnInit {
 
     this._manualListeners.forEach((listener, event) => element.addEventListener(event, listener));
 
-    _focusMonitor.monitor(_elementRef).pipe(takeUntil(this._destroyed)).subscribe(origin => {
+    _focusMonitor.monitor(_elementRef, true).pipe(takeUntil(this._destroyed)).subscribe(origin => {
```

Comment:
This feels like it could case some unintended behavior. E.g. if the tooltip is set on a container with a bunch of content, this will cause it to be shown when any of its descendants are focused.